fsh-tank/input/fsh/CervicalCancerManagementActivity.fsh

```diff
@@ -21,10 +21,9 @@ Usage: #definition
 * intent = $RINTENT#proposal "Proposal"
 * priority = $RPRIOR#routine "Routine"
 * dynamicValue[+].path = "orderDetail" // this is the ServiceRequest.orderDetail element
-* dynamicValue[=].expression.language = $EXLANG|4.0.1#text/cql "CQL"
+* dynamicValue[=].expression.language = $EXLANG|4.0.1#text/cql-identifier "CQL"
 * dynamicValue[=].expression.expression = "ServiceRequestOrderDetail"
-* dynamicValue[=].expression.reference = "Library/ManagementLibrary|1.0"
 * dynamicValue[+].path = "occurrenceDateTime" // this is the ServiceRequest.occurrence element
-* dynamicValue[=].expression.language = $EXLANG|4.0.1#text/cql "CQL"
+* dynamicValue[=].expression.language = $EXLANG|4.0.1#text/cql-identifier "CQL"
 * dynamicValue[=].expression.expression = "OrderDate"
 * dynamicValue[=].expression.reference = "Library/ManagementLibrary|1.0"
```

Comment:
There are several instances where the `expression.reference` is removed from one expression element but not from the secondary one. Is this intentional?